### DISCUSSION TITLE
v1.43.9 — First class ORM support in Encore.ts

### DISCUSSION THREAD

**Discussion by marcuskohlberg** [encoredev/encore#1560] [General]
In this release we've added greatly improved support for using ORMs and migration frameworks with Encore.ts.
**Remember to update**: `encore version update`

Since Encore’s approach to database management is flexible, i.e. it uses standard SQL migration files, it now supports integration with most popular ORMs.

📚 We've created examples and docs for how to use:
- [Prisma](https://encore.dev/docs/ts/develop/orms/prisma)
- [Drizzle](https://encore.dev/docs/ts/develop/orms/drizzle)
- [Knex.js](https://encore.dev/docs/ts/develop/orms/knex)
- [Sequelize](https://encore.dev/docs/ts/develop/orms/sequelize)

Many other ORMs are also supported. Here's how you can reason about it:
- **ORM Compatibility**: If an ORM can connect to a database via a standard SQL driver, it will work with Encore.
- **Migration Tool Compatibility**: If your migration tool generates SQL migration files without additional customization, it can be used with Encore.

[Learn more in the ORM docs](https://encore.dev/docs/ts/develop/orms)

## Other improvements & Bugfixes
* runtimes/core: handle missing auth schema by @eandre in https://github.com/encoredev/encore/pull/1541
* runtimes/js: better handling of void returns by @eandre in https://github.com/encoredev/encore/pull/1543
* make it possible to override what rust builder to use by @fredr in https://github.com/encoredev/encore/pull/1551
* Tidy mods by @ekerfelt in https://github.com/encoredev/encore/pull/1558

**Full Changelog**: https://github.com/encoredev/encore/compare/v1.43.7...v1.43.9

<hr /><em>This discussion was created from the release <a href='https://github.com/encoredev/encore/releases/tag/v1.43.9'>v1.43.9 — First class ORM support in Encore.ts</a>.</em>

**Comment by salmgazer** [comment]
When using an ORM such as drizzle, is encore able to record db calls and show on the encore dashboard what queries hit the database per request?

**Reply by marcuskohlberg** [reply]
Hi, great question.
Not currently, but full tracing support for ORMs is coming soon.

**Reply by stdlout** [reply]
@marcuskohlberg Is there I plan for this?  I can't seem to find anything about it?

**Reply by blackirishmec** [reply]
Pinging for a follow up, esp in relation to Prisma!
